extide/gradle/src/org/netbeans/modules/gradle/customizer/GradleExecutionPanel.java

```diff
@@ -37,7 +42,10 @@
             + "automatically trusted.</p>",
 })
 public class GradleExecutionPanel extends javax.swing.JPanel {
-
+    private static final String LEVEL_PERMANENT = "Permanent"; // NOI18N
```

Comment:
Well, this is a bit ugly '90-es style.
Let's have an enum Level { PERMANENT, TEMPORARY, NONE } instead.

Reply:
> Well, this is a bit ugly '90-es style.

a matter of style, but indeed should adhere to the module's overall style. Will enum-erate.

Reply:
Redone in 7e035b7